Framework/Kernel/inc/MantidKernel/ITimeSeriesProperty.h

```diff
@@ -70,6 +73,9 @@ class ITimeSeriesProperty {
   // Returns whether the time series has been filtered
   virtual bool isFiltered() const = 0;
 
+  // Remove time series datapoints with times outside of TimeROI
```

Comment:
this is not strictly true since we're leaving two values in each masked region.  This should be stated

Reply:
Done.